.github/workflows/release.yaml

```diff
@@ -25,6 +25,27 @@ jobs:
           command: run
           args: --example schema
 
+      - name: Generate Rust Docs
+        uses: actions-rs/cargo@v1
+        with:
+          command: doc
+          args: --no-deps
+
+      - name: Stage Cargo Docs
+        run: |
+          # Add redirect page to inner doc index
+          echo "<meta http-equiv=\"refresh\" content=\"0; url=funding_trading_bridge_smart_contract\">" >> ./target/doc/index.html
+          # Create doc deployment location
+          mkdir ./pages-files
+          # Move documentation to its configured location in settings
+          cp -r target/doc ./pages-files/docs
+
+      - name: Deploy Docs to GitHub Pages
+        uses: JamesIves/github-pages-deploy-action@v4.2.2
+        with:
+          branch: gh-pages
```

Comment:
After this merges, I'll need to make a `gh-pages` branch and submit a ticket to make it protected from deletion